components/centraldashboard/public/assets/i18n/languages.json

```diff
@@ -59,12 +59,17 @@
             "notebookCard.errRetrievingNotebooks": "Error retrieving Notebooks",
             "notebookCard.errNoNotebooks": "No Notebooks in namespace {namespace}",
             "notebookCard.txtAccessed": "Accessed ",
-            "registrationPage.lblWelcome": "Welcome",
-            "registrationPage.registrationDesc": "In order to use Kubeflow, a namespace for your account must be created. Follow the steps to get started",
-            "landingPage.landingDesc": "If you have questions or issues, please contact our Slack support channel at",
-            "landingPage.slackLink": "https://statcan-aaw.slack.com",
-            "landingPage.firstTime": "This seems to be the first time you access Kubeflow, we are happy to see you!",
-            "landingPage.autoNamespace": "Your namespace is automatically generated.",
+            "notebookDefaultCard.connect": "Connect",
+            "notebookDefaultCard.create": "Create",
+            "notebookDefaultCard.details": "Details",
+            "notebookDefaultCard.name": "Name",
+            "notebookDefaultCard.type": "Type",
+            "notebookDefaultCard.helpText": "The default notebook is one that will automatically start upon login into Kubeflow",
```

Comment:
to me is "is one" sounds kind of weird. Would "The default notebook will automatically start upon login in kubeflow" be better?